### DISCUSSION TITLE
Can't use modern websites on old guest MacOS ("security failure the server reply is invalid")

### DISCUSSION THREAD

**Discussion by lucaspukus** [utmapp/UTM#4198] [General]
Hi.
Ive installed MacOS 9.2 with UTM on an M1 (pro) MBP.
When using internet explorer im only able to access the google webpage. Any other page I try (including apple) or any links on the google results I get give me the following error "security failure the server reply is invalid". Also happens on netscape. Is there a fix or no?
Im trying to install a couple of old games & cant seem to share the files into the VM. Seems I can only share "iso" or "IMG" files, so I wanted to try downloading them (to no avail). 
Can anyone help with either issue.
The files I want to share are "Hellcats over the pacific.SIT" & "The secret of monkey island"
TIA

**Comment by conath** [comment]
Old web browsers are not compatible with the modern web, especially regarding encryption. Hence why you're seeing those "security failure" errors. For a retro web browsing experience I'd recommend ["the old net"](http://theoldnet.com/).

To get files to a MacOS 9 guest, make a folder containing the files and convert it to ISO. Then you can insert it into the VM as a CD. I made a macOS Service for converting folder to ISO. It's available on [my UTM website](https://chrisp.cafe/UTM/) (near the bottom of the page).

**Reply by lucaspukus** [reply]
Worked a treat!. Cheers!. any sites you would recommend for downloading old files?
